controllers/apply/form-router-next/steps.js

```diff
@@ -237,24 +254,7 @@ module.exports = function(formId, formBuilder) {
                         }
                     }
 
-                    let redirectUrl;
-                    if (req.body.previousBtn) {
-                        const { previousPage } = form.pagination({
-                            baseUrl: res.locals.formBaseUrl,
-                            sectionSlug: req.params.section,
-                            currentStepIndex: stepIndex,
-                            copy: res.locals.copy
-                        });
-                        redirectUrl = previousPage.url;
-                    } else {
-                        const { nextPage } = form.pagination({
-                            baseUrl: res.locals.formBaseUrl,
-                            sectionSlug: req.params.section,
-                            currentStepIndex: stepIndex,
-                            copy: res.locals.copy
-                        });
-                        redirectUrl = nextPage.url;
-                    }
+                    const redirectUrl = determineRedirectUrl();
```

Comment:
You can inline references to this variable, call the method where it is used instead.

Reply:
Did you mean directly call the method when needed instead of creating a variable `redirectUrl` ?